cl/search/models.py

```diff
@@ -313,10 +313,11 @@ def pacer_url(self):
         from cl.lib.pacer import map_cl_to_pacer_id
         court_id = map_cl_to_pacer_id(self.court.pk)
         if self.court.jurisdiction == Court.FEDERAL_APPELLATE:
-            return (u'https://ecf.%s.uscourts.gov/n/beam/servlet/TransportRoom?'
+            return (u'https://ecf.%s.uscourts.gov'
```

Comment:
Can we make this a `#noqa` instead of breaking the URL?

Reply:
I feel better about breaking on a ? than on the path, but I leave this to your call. I didn't notice that it was breaking before, but now it's jarring to me.